public/styles.css

```diff
@@ -13,19 +13,21 @@ nav {
     font-size: 20px;
     font-weight: 700;
     position: relative;
+    z-index: -100;
+    background: #333;
```

Comment:
Added the background color back in, but positioned the nav below its pseudo elements (`z-index: -10`) and the remaining content (`z-index: 0` (although technically `auto`)).

I used `-100` here as z-indexing by factors of `10` gives you some wiggle room later on.

Note: `z-index` only works if the element has `position: relative` or `position: absolute`... an issue a often run into.